test/scsb_rest_client.test.js

```diff
@@ -1,13 +1,55 @@
 /* eslint no-new:0*/
```

Comment:
Do you have a strong preference for assert vs should vs expect?
Can we keep it to one style?
My preference is `should()` or `expect()` - does one style get particularly nasty when dealing with promises / chai-as-promised?